code/src/components/ThoughtsList.css

```diff
@@ -0,0 +1,63 @@
+.thoughts-list-wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 35px;
+}
+
+.single-thought-div {
+    border: 1px solid black;
+    width: 400px;
+    height: 150px;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    word-wrap: break-word;
+    justify-content: space-between;
+    background-color: white;
+    box-shadow: 5px 5px 0px 0px black;
+}
+
+.single-thought-div.fade-in {
+    opacity: 0;
+    animation: fade-in 0.9s ease forwards;
+  }
+
+  @keyframes fade-in {
+    from {
+      opacity: 0;
+    }
+    to {
+      opacity: 1;
+    }
+  }
+
+.heart-emoji {
+    background-color: rgb(255, 204, 212);
+    padding: 10px 10px 10px 10px;
```

Comment:
Since you have the same value of pixels all around. You could just write: padding: 10px and it will apply to top, bottom left and right.